### PR TITLE
Fix spacing diffs from downstream

### PR DESCRIPTION
**Pull request by machadovilaca**
**What this PR does / why we need it**:

Fix spacing diffs from downstream

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/runbooks/OutdatedVirtualMachineInstanceWorkloads.md [hunk 1]
```diff
@@ -68,9 +68,9 @@ If a VMI is not live-migratable and if `runStrategy: always` is set in the
 corresponding `VirtualMachine` object, you can update the VMI by manually
 stopping the virtual machine (VM):
 
-```bash
-$ virctl stop --namespace <namespace> <vm>
-```
+  ```bash
+  $ virctl stop --namespace <namespace> <vm>
```
Comment (ousleyp) [inline docs/runbooks/OutdatedVirtualMachineInstanceWorkloads.md:72]: Not related to this PR, but there is a typo in `virtctl` (spelled `virctl` currently)

**Review by ousleyp — APPROVED** [review]
@machadovilaca Your changes LGTM! I wasn't sure about the varying amounts of spaces at first, but I see that it's exactly the same in openshift/runbooks. I'm glad that github flavored markdown is forgiving :)

I did notice one small error in the existing code that we need to fix in all 3 runbook locations, but that can wait.

File: docs/runbooks/OutdatedVirtualMachineInstanceWorkloads.md (hunk 1 above)
Comment (machadovilaca) [reply docs/runbooks/OutdatedVirtualMachineInstanceWorkloads.md]: updated

**Comment by machadovilaca** [comment]
> @machadovilaca Your changes LGTM! I wasn't sure about the varying amounts of spaces at first, but I see that it's exactly the same in openshift/runbooks. I'm glad that github flavored markdown is forgiving :)
> 
> I did notice one small error in the existing code that we need to fix in all 3 runbook locations, but that can wait.

Yes, I wondered if I should go for a standard number of spaces, or follow what is in openshift/runbooks, but I thought for the scope of this PR this made more sense

**Review by sradco — APPROVED** [review]

**Pull request merged**